src/textbooks/textbook-card/TextbooksCard.jsx

```diff
@@ -0,0 +1,146 @@
+import PropTypes from 'prop-types';
+import { useContext, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  ActionRow,
+  Card,
+  Collapsible,
+  Icon,
+  IconButtonWithTooltip,
+  useToggle,
+} from '@openedx/paragon';
+import {
+  EditOutline as EditIcon,
+  RemoveRedEye as ViewIcon,
+  DeleteOutline as DeleteIcon,
+} from '@openedx/paragon/icons';
+import { AppContext } from '@edx/frontend-platform/react';
+
+import DeleteModal from '../../generic/delete-modal/DeleteModal';
+import { RequestStatus } from '../../data/constants';
+import { getCurrentTextbookId, getSavingStatus } from '../data/selectors';
+import TextbookForm from '../textbook-form/TextbookForm';
+import { getTextbookFormInitialValues } from '../utils';
+import messages from './messages';
+
+const TextbookCard = ({
+  textbook,
+  courseId,
+  handleSavingStatusDispatch,
+  onEditSubmit,
+  onDeleteSubmit,
+  textbookIndex,
+}) => {
+  const intl = useIntl();
+  const { config } = useContext(AppContext);
+
+  const savingStatus = useSelector(getSavingStatus);
+  const currentTextbookId = useSelector(getCurrentTextbookId);
+
+  const [isTextbookFormOpen, openTextbookForm, closeTextbookForm] = useToggle(false);
+  const [isDeleteModalOpen, openDeleteModal, closeDeleteModal] = useToggle(false);
+
+  const { tabTitle, chapters, id } = textbook;
+
+  const onPreviewTextbookClick = () => {
+    window.location.href = `${config.LMS_BASE_URL}/courses/${courseId}/pdfbook/${textbookIndex}/`;
```

Comment:
I think a better user experience would be to have the preview open in a new tab.

Reply:
Fixed

Reply:
Fixed